zoo-css-challenge/style.css

```
@@ -108,28 +117,65 @@ footer {
   border: 5px solid #747c92;
 }
 
+/*2) Bears */
+/* images vertically aligned to the top of the container*/
 .image-container img:first-of-type {
-  vertical-align: bottom;
+  vertical-align: top;
+  border: 5px solid #617fd3;
+}
+
+.image-container img:nth-of-type(2) {
+  vertical-align: top;
+  border: 5px solid #6b0579;
 }
 
+.image-container img:nth-of-type(3) {
+  vertical-align: top;
+  border: 5px solid #e2500c;
+}
+
+
+/* End 2*/
+
+
 .tigers {
   background-color: #0b3c49;
   color: #f0e100;
 }
 
+/* 3) Tigers */
+.tigers h2, h3{
+  color:#FFD500 ;
+}
+
+/* End 3) Tigers */
+
+/* 4) Giraffes*/
 .giraffe {
-  background-color: #465775;
-  color: #56e39f;
+  background-color: #483C46;
+  color: #BEEE62;
 }
 
 .giraffe h2 {
-  color: #ef6f6c;
+  color: #F4743B;
+}
+.giraffe p:first-of-type{
+  font-size:20px;
```

```suggestion
  font-size: 1.25rem;
```
We should use relative units where possible, especially for `font-size`